Example/YelpAPI.xcodeproj/project.pbxproj

```diff
@@ -124,6 +128,8 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				0B5080851CBEEA950071508E /* OHHTTPStubs.framework in Frameworks */,
+				0B5080831CBEEA800071508E /* OCMock.framework in Frameworks */,
```

Comment:
Aha I found the issue! The Podfile needs to be changed from "YelpAPI_Tests" to "YelpAPI_PodsTests", it's integrating into the wrong target.

``` diff
diff --git a/Example/Podfile b/Example/Podfile
index f6136b1..ac0fc36 100644
--- a/Example/Podfile
+++ b/Example/Podfile
@@ -6,7 +6,7 @@ target 'YelpAPI_Example', :exclusive => true do
   pod "YelpAPI", :path => "../"
 end
 
-target 'YelpAPI_Tests', :exclusive => true do
+target 'YelpAPI_PodsTests', :exclusive => true do
   pod 'TDOAuth', '~> 1.1'
   pod 'OCMock', '~> 3.1.5'
   pod 'OHHTTPStubs', '4.6.0'
```